plexapi/collection.py

```diff
@@ -226,6 +257,39 @@ def sortUpdate(self, sort=None):
             raise BadRequest('Unknown sort dir: %s. Options: %s' % (sort, list(sort_dict)))
         self.editAdvanced(collectionSort=key)
 
+    def visibleUpdate(self, library=None, home=None, shared=None):
+        """ Update the visible on setting.
+
+            Parameters:
+                library (bool): Set visible on library.
+                home (bool): Set visible on Home.
+                shared (bool): Set visible on Shared Users' Home.
+
+            Example:
+
+                .. code-block:: python
+
+                    collection.visibleUpdate(library=True, home=False)
+        """
+        if library is None and home is None and shared is None:
```

Comment:
No need to check for `None`. The request can go through without changing anything.